ime/app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardSuggestions.java

```diff
@@ -621,6 +621,10 @@ protected void handleCharacter(
                     cursorPosition = -1;
```

Comment:
This seems problematic. maybe it should be set to `mGlobalCursorPosition`?

Reply:
I'm not too sure about that... `cursorPosition` is made negative on purpose because the code works differently at the end of a word than at the middle. Presumably it's faster at the end, I didn't write that part.

(**BTW!** I pushed another commit to move my fix somewhere else, because I mistakenly overwrote `mGlobalCursorPosition` to -1 sometimes. It surprisingly didn't change anything because the code depending on that variable had no chance to run then, but this way it's written to less times and raises fewer questions.)